lib/index.js

```diff
@@ -27,6 +27,17 @@ class PWMetrics {
     this.sheets = opts.sheets;
     this.expectations = opts.expectations || false;
     this.tryLighthouseCounter = 0;
+    this.desiredMetrics = [];
+    if (this.flags.desiredMetrics) {
+      this.desiredMetrics = this.flags.desiredMetrics.filter(metric => {
+        if (!Object.values(metrics.ids).includes(metric)) {
+          console.log(getMessageWithPrefix('WARNING', 'UNKOWN_METRIC', metric));
+          return false;
+        }
+        return true;
+      });
+    }
+    this.desiredMetrics = this.desiredMetrics.length ? this.desiredMetrics : ['tti', 'psi', 'ttfmp', 'ttfcp'];
```

Comment:
Yeap, sorry, vice versa``this.desiredMetrics = Object.assign([], ['tti', 'psi', 'ttfmp', 'ttfcp'], this.flags.desiredMetrics);`